### PR TITLE
Improve exception handling while cleaning up mysql cleanup thread

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

**Comment by bdmogal** [comment]
Verified that error log does not show up when running with HSQLDB

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/DBUtils.java [hunk 1]
```diff
@@ -85,12 +85,20 @@ private static void shutDownMySQLAbandonedConnectionCleanupThread(ClassLoader cl
...
     }
     try {
-      Class<?> mysqlCleanupThreadClass = classLoader.loadClass("com.mysql.jdbc.AbandonedConnectionCleanupThread");
+      Class<?> mysqlCleanupThreadClass;
+      try {
+        mysqlCleanupThreadClass = classLoader.loadClass("com.mysql.jdbc.AbandonedConnectionCleanupThread");
+      } catch (ClassNotFoundException e) {
+        // Ok to ignore, since we may not be running mysql
+        LOG.trace("Failed to load MySQL abandoned connection cleanup thread class. Presuming DB Adapter is " +
+                    "not being run with MySQL and ignoring");
```
Comment (chtyim) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/DBUtils.java]: Log the "e" as well.

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/DBUtils.java [hunk 2]
```diff
@@ -85,12 +85,20 @@ private static void shutDownMySQLAbandonedConnectionCleanupThread(ClassLoader cl
...
+        return;
+      }
       Method shutdownMethod = mysqlCleanupThreadClass.getMethod("shutdown");
       shutdownMethod.invoke(null);
-      LOG.info("Successfully shutdown MySQL connection cleanup thread.");
+      LOG.debug("Successfully shutdown MySQL connection cleanup thread.");
     } catch (Throwable e) {
-      // cleanup failed, ignoring silently
+      // cleanup failed, ignoring silently with a log, since not much can be done.
```
Comment (chtyim) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/DBUtils.java]: Log with the "e" as well.

**Comment by chtyim** [comment]
LGTM

**Comment by bdmogal** [comment]
This does not affect any unit test. Not going to run a build to hog resources on the build server. Squashing and merging.

**Pull request merged**